docs/build-windows.md

```diff
@@ -2,16 +2,25 @@
 
 Windows builds can be created using:
 
-- MSVC compiler, Visual Studio 2022 IDE suite, and vcpkg to provide dependencies
-  *(recommended)*.
+- MSClang compiler, Visual Studio 2022 IDE suite, and vcpkg to provide
+dependencies. This is the fully-supported toolchain used to
+create release builds.
 - The Clang or GCC compilers using the Meson buildsystem running within the
   MSYS2 environment to provide dependencies.
 
-
 ## Build using Visual Studio
 
 1. Install Visual Studio Community 2022: <https://visualstudio.microsoft.com/>.
-2. Install vcpkg: <https://github.com/Microsoft/vcpkg#quick-start-windows>.
+    - Select "C++ Clang tools for Windows" in the Visual Studio installer
+2. Install/configure vcpkg.
+    - Recent versions of Visual Studio already include it[^1].
+    - Existing [standalone versions](<https://github.com/Microsoft/vcpkg#quick-start-windows>) should work as well.
+    - For either option, run the following command to integrate with Visual Studio:
+
+        ``` shell
+        vcpkg integrate install
```

Comment:
Do you have to be in one of the "Native Tools Command Prompt for VS" shells to run this with the vcpkg included with VS? Or does any shell work?

Reply:
Yes, you need a Developer Command prompt like any other VS CLI tool, it doesn't put any of its directories on the system path by default AFAIK.

Reply:
I would mention that in the doc then. Otherwise people using VS for the first time might not be aware.

Reply:
Good catch @shermp, we should mention this.

Reply:
Since the standalone installation instructions clearly indicate you need to run that command, I'll amend it just for the VS-installed version, good suggestion.